src/Uno.UI/UI/Xaml/Controls/MediaPlayerElement/MediaTransportControls.MediaPlayer.cs

```diff
@@ -459,13 +459,25 @@ private void OnVolumeChanged(object sender, RangeBaseValueChangedEventArgs e)
 		{
 			if (_mediaPlayer is not null)
 			{
-				_mediaPlayer.Volume = e.NewValue;
+				_mediaPlayer.Volume = e.NewValue / 100.0;
```

Comment:
This will affect iOS and Android too. I see you made adjustments only to MacOS, Win32/X11 (the shared file), and WASM.

Reply:
I only found Wasm, MacOS and the shared LibVLC based implementation. I assumed the shared extension is used by Android and iOS as well. Searching again, I did not find any other implementations.

Can you point me to where they are?

Reply:
Nevermind, I just found them. They do not use the extension mechanism, which is why I missed them. I will apply the change there as well.